back-end/cmp/celery.py

```diff
@@ -0,0 +1,10 @@
+import os
+from celery import Celery
+from dotenv import load_dotenv
+
+load_dotenv()
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", f"cmp.settings.{os.getenv('SETTINGS')}")
+app = Celery("cmp")
+app.config_from_object("django.conf:settings", namespace="CELERY")
```

Comment:
add config variables in settings e.g `CELERY_BROKER_URL=amqp://guest:guest@rabbitmq:5672` for rabbitmq message broker. All Celeray associated configs should start with `CELERY_` 

In addition to that, Celery should be able to log into  `logs/celery.log`

Reply:
resolved